### PR TITLE
W-18861949-ListOfStringsRN-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 1]
```diff
@@ -4,6 +4,29 @@ Use MuleSoft Composer to build connected experiences faster. You don't need to w
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
+== October 12, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* When looping over a selected List of Strings in a For Each loop, you can now use a single string value across multiple cards in the loop.
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:11]:
```suggestion
* When looping over a selected list of strings in a for each loop, you can now use a single string value across multiple cards in the loop.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 2]
```diff
@@ -4,6 +4,29 @@ Use MuleSoft Composer to build connected experiences faster. You don't need to w
...
+
+==== Composer Platform
+
+[%header%autowidth.spread]
+
+|===
+
+|Known Issue|Workaround |ID
+
+|When deleting a connection, Composer ensures that no flows use that connection. If flows use that connection, they appear in a list in the *Delete Connection* window. However, in rare cases, flows created or edited before July 12, 2022 might not appear in the list. | |
```
Comment (IsaacEldridge) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:23]:
```suggestion
|When deleting a connection, Composer ensures that no flows use that connection. If flows use that connection, they appear in a list in the *Delete Connection* window. However, in rare cases, flows created or edited before July 12, 2022 might not appear in the list. | n/a |
```

Comment (IsaacEldridge) [reply en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:23]: I don't know if it's better to do that or just leave it blank

File: en/salesforce/src/docs/ms_composer_release_notes_2022.adoc [hunk 3]
```diff
@@ -9,6 +9,36 @@ New versions are automatically available once they are released; you don't need
 //| Fixed an issue in NetSuite Connector in which nested fields containing the `list` type failed. | COCN-726
 //* Composer now supports the static creation of lists. You can now dynamically complete fields that are enabled for lists even if those fields do not have an existing list to map to in the flow.
 
+== October 12, 2022
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* When looping over a selected List of Strings in a For Each loop, you can now use a single string value across multiple cards in the loop.
```
Comment (IsaacEldridge) [inline en/salesforce/src/docs/ms_composer_release_notes_2022.adoc:16]:
```suggestion
* When looping over a selected list of strings in a for each loop, you can now use a single string value across multiple cards in the loop.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc (hunk 2 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:23]: I will do this at another time - this appears throughout the entire topic.

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**